src/Nethermind/Nethermind.HealthChecks/NodeHealthService.cs

```diff
@@ -110,6 +142,50 @@ public CheckHealthResult CheckHealth()
                    _healthHintService.MaxSecondsIntervalForProducingBlocksHint();
         }
 
+        public bool CheckClAlive()
+        {
+            if (_blockFinder.Head!.Difficulty == UInt256.Zero)
```

Comment:
this logic is wrong. Imagine your node is syncing. Your Head = 0, but you will receive fcUs and newPayloads. If you have finalTotalDifficulty (PosSwitcher.FinalTotalDifficulty) (so the merge happened you can check for fcU and newPayloads only. In other cases please check all three methods: fcu, newPayload, exchangeTransition